docs/administration/configuring-replication/create-replication-rules.md

```diff
@@ -47,7 +47,11 @@ A replication endpoint must exist before you create a replication rule. To creat
 
    If you do not enter a namespace, resources are placed in the same namespace as in the source registry. 
 
-   ![Destination and namespaces](../../../img/replication-rule4.png)
+   ![Destination and namespaces](../../img/replication-rule4.png)
+   
+   **NOTE:** Because there's some API change in 2.0 release in order to support [OCI](https://github.com/opencontainers/distribution-spec).
+   You **can not** replication from harbor 1.x to 2.0, and you **can not** replication artifacts with **manifest list** from 2.0 to 1.x. 
```

Comment:
change to "replication from Harbor v1.x to v2.0 is not officially supported, and replicating any artifacts like manifest lists or CNABs leveraging OCI index is not possible"